standards/technical-approach.md

```diff
@@ -1,12 +1,30 @@
 # Technical Approach Documents
 
-Technical Approach Documents (TADs) serve to help the engineering team document solutions to feature development, share ideas between teams and help highlight potential issues with tasks before they are encountered during the development process.
+Technical Approach Documents (TADs) serve to help the engineering team document solutions to feature development, share ideas between teams, and help highlight potential issues with tasks before they are encountered during the development process.
 
-TADs should strive to describe the whats, hows and whys of the technical problem that is trying to be solved. It should strive to understand the problem at hand in a comprehensive way and explore any challenges so that engineers can be confident in their implementations of solutions. In short writing a TAD should answer any technical questions prior to the start of actual development work, freeing up developers to focus on implementation once the major questions have been resolved.
+TADs should strive to describe the whats, hows and whys of the technical problem that is trying to be solved. It should strive to understand the problem at hand in a comprehensive way and explore any challenges so that engineers can be confident in their implementations of solutions. In short, writing a TAD should answer any technical questions prior to the start of actual development work, freeing up developers to focus on implementation once the major questions have been resolved.
+
+## Audience
+
+TADs are ultimately technical documents for a technical audience, not for Product or non-technical stakeholders. They do require approval from the Engineering Leadership Team (ELT) as [described below](#review-process).
```

Comment:
one of the main challenges I encountered when I wrote TADs is figuring out the audience I was writing to. 

My initial assumption was that I was writing for my peers, so in my case iOS developers. So I would not add in my TAD explanations about certain iOS technologies or why other things are ok or not ok to use, because they are obvious in iOS development. But then eventually I had to rewrite entire paragraphs to make them understandable to engineers not familiar with the iOS stack. I have no problem in doing that btw: my point here is that saying "technical audience" is kind of vague and may include people with completely different technical training.

I know that "technical audience" includes engineers that will implement the feature, so in my example, iOS engineers. But that's not the only audience. What can we assume for this other audience?

Reply:
I agree it would be helpful to have more clarity on this. Is ELT different from DLT? Maybe ELT should be defined.